model/license.go

```diff
@@ -218,6 +220,10 @@ func (f *Features) SetDefaults() {
 	if f.AdvancedLogging == nil {
 		f.AdvancedLogging = NewBool(*f.FutureFeatures)
 	}
+
+	if f.Cloud == nil {
+		f.Cloud = NewBool(*f.FutureFeatures)
```

Comment:
Same comment here as on the other, probably want to default to `false`

Reply:
Done